### ISSUE TITLE
getLicense functions fails to handle some incorect licenses fields in package.json

### ISSUE THREAD

**Issue by drahoslove** [google/js-green-licenses#104]
https://github.com/google/js-green-licenses/blob/e84fec11b5c55e3984798e1452c85caf830d521b/src/checker.ts#L192

This function will return `"()"` for both
```json
...
licenses: [  ],
...
```
and
```json
...
licenses: [ "MIT", "BSD"  ],
...
```
I think it should return `null` for the first case and `"( MIT OR BSD )"` for the other.

**Comment by kjin** [comment]
Agree for `licenses: []` but the [package.json docs](https://docs.npmjs.com/files/package.json#license) don't mention `licenses` as a list of strings. Unless you know of packages that do exactly that (and shows up correctly on the npm web UI), maybe that should return `null` as well.

**Comment by drahoslove** [comment]
@kjin, for example, this one:
https://www.npmjs.com/package/rc/v/1.0.1
https://registry.npmjs.org/rc/1.0.1

**Comment by kjin** [comment]
https://github.com/dominictarr/rc/blob/a97f6adcc37ee1cad06ab7dc9b0bd842bbc5c664/package.json#L14

Not to get too nit-picky but isn't the field in question `"license": "(BSD-2-Clause OR MIT OR Apache-2.0)"` and not `"licenses": ["BSD-2-Clause", "MIT", "Apache-2.0"]`?

**Comment by kjin** [comment]
Never mind, you're right: https://www.npmjs.com/package/kjin-license-test

**Comment by ofrobots** [comment]
I think it would be better to have a 'safe' default for non-standard / un-parsable license fields.

**Comment by JustinBeckwith** [comment]
The empty array part was addressed in #146, and from reading the thread - I think we're going to keep the stricter package json parsing as is.
